packages/expo-modules-autolinking/src/platforms/ios.ts

```diff
@@ -27,5 +28,46 @@ export async function resolveModuleAsync(
     podName,
     podspecDir,
     flags: options.flags,
+    modulesClassNames: revision.config?.iosModulesClassNames(),
   };
 }
+
+/**
+ * Generates Swift file that contains all autolinked Swift packages.
+ */
+export async function generatePackageListAsync(
```

Comment:
I can't change it now because it's part of the interface of platform-specific implementation that is run on `expo-modules-autolinking generate-package-list` command. Also, we don't generate files for Objective-C, so Swift will be the default and only one 😉